packages/cli/src/commands/setup/auth/auth.js

```diff
@@ -122,13 +132,90 @@ export async function builder(yargs) {
       'Set up auth for for SuperTokens',
       (yargs) => standardAuthBuilder(yargs),
       async (args) => {
-        const handler = await getAuthHandler(
+        const { handler } = await getAuthSetup(
           '@redwoodjs/auth-supertokens-setup'
         )
         console.log()
         handler(args)
       }
     )
+    .strict(false)
+    .parserConfiguration({ 'unknown-options-as-args': true })
+    .command(
+      '$0 <npm-package>',
+      'Generate an auth configuration for the given provider',
+      () => {},
+      async () => {
+        // Here be dragons...
+        // There's no way for the user to actually end up here, but we need
+        // this command to get the help output we want from yargs
+      }
```

Comment:
The help still seems to show when I remove the builder and handler here (`() => {}` and `async () => {}`); doesn't seem to matter if we leave them in or not, but the `.command()` is definitely necessary. But maybe we could move the comment up to before `.command()` to reflect that?